core/payments/clientledger/client_ledger.go

```diff
@@ -154,6 +154,8 @@ func (cl *ClientLedger) debitReservationOnly(
 		panic(fmt.Sprintf("reservation debit failed: %v", err))
 	}
 
+	cl.accountantMetricer.RecordReservationPayment(cl.accountID.Hex(), remainingCapacity)
```

Comment:
nit: could it make sense to do this inside reservationLedger.Debit() instead?

Reply:
Since these metrics are for clients only, they cannot be pushed down into the `ReservationLedger`, which will also run on dispersers and validators